tests/newblob/constraint.sy

```diff
@@ -0,0 +1,17 @@
+Foo :: newblob {
```

Comment:
Add a test where the newblob is passed as arguments. One as generic, and one where the type is specified.